packages/block-library/src/calendar/index.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Server-side rendering of the `core/calendar` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/calendar` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content.
+ */
+function render_block_core_calendar( $attributes ) {
+	global $monthnum, $year, $post;
+	$previous_monthnum;
+	$previous_year;
```

Comment:
Unlike JavaScript, variables in PHP are scoped **locally by default**. These two lines are therefore unnecessary.

For example, this will output 'example' only once:

```php
function example() {
	$foo = 'example';
	echo $foo; // Will output 'example'
}
example();
echo $foo; // Will output nothing
```